providers/twilio/src/lib/twilio.provider.ts

```diff
@@ -36,4 +38,63 @@ export class TwilioSmsProvider implements ISmsProvider {
       date: twilioResponse.dateCreated.toISOString(),
     };
   }
+
+  getMessageId(body: any | any[]): string[] {
+    if (Array.isArray(body)) {
+      return body.map((item) => item.MessageSid);
```

Comment:
Will this be the same as here db8617cd21811d4849154e/providers/twilio/src/lib/twilio.provider.ts#L37 ?

Reply:
I believe not.
Below is the webhook body that Twilio sends, so based on it, it wouldn't be same as on like 37 (line 37 is the API response for sending messages I believe)
```
"AccountSid": "ACxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
"From": "+15017250604"
"MessageSid": "SM1342fe1b2c904d1ab04f0fc7a58abca9"
"MessageStatus": "sent"
"SmsSid": "SM1342fe1b2c904d1ab04f0fc7a58abca9"
"SmsStatus": "sent"
```

Reply:
Okay that needs to be the case otherwise we will not be able to identify for what email the webhook is sent for...
Can we send a custom prop to Twillio and also get back in the webhook as well?
Or is there a prop that we get both as response to sending the sms and in the webhook body?

Reply:
To be completely honest, I will need to check about it in detail for myself once.
I will check it and get back to you. 🙇

Reply:
Hi @davidsoderberg 👋 
Sorry for the delay.

I went through the docs and it seems that the value of `MessageSid` from the webhook response is most likely the same as `sid` from the response while sending the message

Reply:
ok then we can go with that 😄